### PR TITLE
Feature/us8772 batch deposit trip transfers

### PR DESCRIPTION
**Pull request by andycanterbury**
This PR adds a feature to group trip transfers by month and create a batch and a deposit for them which is already marked as exported so that Finance doesn't have to see them.

### CODE REVIEW

File: MinistryPlatform/Stewardship/BatchRepository.cs [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+    {
+        private readonly IApiUserRepository _apiUserRepository;
+        private const string AppCode = "CRDS-COMMON";
+
+        private readonly int _batchEntryType;
+
+        public BatchRepository(IMinistryPlatformRestRepository ministryPlatformRestRepository, IConfigurationWrapper configurationWrapper, IApiUserRepository apiUserRepository) : base(ministryPlatformRestRepository, configurationWrapper)
+        {
+            _apiUserRepository = apiUserRepository;
+            _batchEntryType = ConfigurationWrapper.GetMpConfigIntValue(AppCode, "TripTransferEntryType") ?? 13;
```
Comment (jpc11623) [inline MinistryPlatform/Stewardship/BatchRepository.cs]: Magic number needs to be moved to a constant value

File: crds-mp-tools/crds-mp-tools.csproj [hunk 2]
```diff
@@ -23,7 +23,7 @@
     <None Include="wwwroot\js\crds-mp-tools.js" />
   </ItemGroup>
   <ItemGroup>
-    <PackageReference Include="Microsoft.ApplicationInsights.AspNetCore" Version="2.1.0" />
+    <PackageReference Include="Microsoft.ApplicationInsights.AspNetCore" Version="2.1.1" />
```
Comment (jpc11623) [inline crds-mp-tools/crds-mp-tools.csproj]: Was this update to the package intended?

File: MinistryPlatform/Stewardship/BatchRepository.cs (hunk 1 above)
Comment (andycanterbury) [inline MinistryPlatform/Stewardship/BatchRepository.cs]: That is a default if we can't get the value from the configuration table. Shouldn't ever get used.

File: crds-mp-tools/crds-mp-tools.csproj (hunk 2 above)
Comment (andycanterbury) [inline crds-mp-tools/crds-mp-tools.csproj]: It was. We are trying to keep the packages up to date in this repo.

File: MinistryPlatform/Stewardship/BatchRepository.cs (hunk 1 above)
Comment (jpc11623) [inline MinistryPlatform/Stewardship/BatchRepository.cs]: I strongly feel that this should be moved to a class-level constant at least.

**Pull request merged**